extension/src/popup/helpers/soroban.ts

```diff
@@ -19,6 +20,11 @@ export enum SorobanTokenInterface {
   mint = "mint",
 }
 
+export const SOROBAN_OPERATION_TYPES = [
+  "invoke_host_function",
+  "invokeHostFunction",
```

Comment:
this is kinda weird that an op type can either be camelCase or snake_case. I'll raise this with the Platform team